packages/theme-chalk/src/tag.scss

```diff
@@ -133,7 +134,7 @@ $tag-icon-span-gap: map.merge(
     padding-right: map.get($tag-icon-span-gap, 'default') - $border-width;
   }
 
-  @each $size in (large, default, small) {
```

Comment:
Remove tag default is safe.

Its default value has always existed in the style above.